src/main/java/at/ac/univie/mminf/luceneSKOS/analysis/engine/jena/SKOSEngineImpl.java

```diff
@@ -227,24 +277,51 @@ public SKOSEngineImpl(InputStream inputStream, String format, List<String> langu
         searcher = new IndexSearcher(DirectoryReader.open(indexDir));
     }
 
+    private static String getHash(Reader reader) throws IOException, NoSuchAlgorithmException
+    {
+        String sha1 = "";
+        char []  cbuf = new char[1024];
+         MessageDigest crypt = MessageDigest.getInstance("SHA-1");
+        crypt.reset();
+        while (reader.read(cbuf) > 0)
+            crypt.update(new String(cbuf).getBytes(StandardCharsets.ISO_8859_1));
```

Comment:
No, you're right. The is no valid reason. I wanted to make a patch as fast as possible for my users who are blocked with the issue. So, I develop it very quicly. I think the code could be improve. Il will think about it and I will propose you improvement suggestions.